waveorder/stokes.py

```diff
@@ -147,16 +147,16 @@ def stokes_after_adr(
     depolarizing retarder (adr) parametrized by its retardance, slow-axis
     orientation, transmittance, and depolarization.
 
-    Note: all four parameters can be array_like, but they must be the same size.
-    If your parameters are in an ndarray with array.shape = (4, ...), use
+    Note: all four parameters can be torch.tensor, but they must be the same size.
```

Comment:
Note that `torch.tensor` is a functional constructor of the `torch.Tensor` type. This is different from `numpy.array` which is effectively an alias.

Reply:
Thanks...changing `torch.tensor` -> `torch.Tensor` in the docstrings.